clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -174,6 +174,27 @@ public class AzureComputeClient extends AzureBaseClient {
     )
   }
 
+  ServiceResponse<Void> disableServerGroup(String resourceGroupName, String serverGroupName) {
+    VirtualMachineScaleSetsOperations ops = getAzureOps(
+      client.&getVirtualMachineScaleSetsOperations, "Get operations object", "Failed to get operation object") as VirtualMachineScaleSetsOperations
+
+    List<String> instanceIds = this.getServerGroupInstances(resourceGroupName,serverGroupName)?.collect {it.resourceId}
+
+    ops.powerOff(resourceGroupName, serverGroupName, instanceIds)
+
+    // TODO: investigate if we can deallocate the VMs
+    //ops.deallocate(resourceGroupName, serverGroupName, instanceIds)
+  }
+
+  ServiceResponse<Void> enableServerGroup(String resourceGroupName, String serverGroupName) {
+    VirtualMachineScaleSetsOperations ops = getAzureOps(
+      client.&getVirtualMachineScaleSetsOperations, "Get operations object", "Failed to get operation object") as VirtualMachineScaleSetsOperations
```

Comment:
When I briefly looked at the JSDK implementation for getting the ops object and I could not guarantee that they don't make a rest call. For instance what if the current token/credentials are expired? Don't they need to renew them as part of this and isn't that another REST call? 

Can you please ask the JSDK guys to confirm that retrieving the operation object is 100% guaranteed not to make a REST call into Azure? If the answer is yes, I will remove that code.